src/python/elrs_helpers.py

```diff
@@ -46,21 +46,31 @@ def get_git_version(env):
     ver = "ver.unknown"
     sha = "000000"
 
-    git_repo = get_git(env)
-    if git_repo:
-        import git
-        sha = git_repo.head.object.hexsha
+    try:
+        call_git = lambda cmd: subprocess.check_output(["git"] + cmd).decode("utf-8").rstrip('\r\n')
+        sha = call_git(["rev-parse", "--short", "HEAD"])
+        ver = call_git(["rev-parse", "--abbrev-ref", "HEAD"])
+        # failure here is tolerable, sometimes HEAD is not tagged
         try:
-            ver = re.sub(r".*/", "", git_repo.git.describe("--all", "--exact-match"))
-        except git.exc.GitCommandError:
+            ver = subprocess.check_call(["git", "describe", "--tags", "--exact-match"])
+        except:
+            pass
```

Comment:
This section would need to be using the same commands and the python git version below. i.e. `--all` and if that fails then symbolic-ref

Reply:
Thanks for the lint, fixed it